packages/react/src/components/Embed/Embed.tsx

```diff
@@ -142,25 +151,27 @@ class Embed extends AutoControlledComponent<WithAsProp<EmbedProps>, EmbedState>
                 controls: false,
                 loop: true,
                 muted: true,
+                poster: placeholder,
                 styles: styles.video,
                 variables: {
                   width: variables.width,
                   height: variables.height,
                 },
               },
             })}
-            {Box.create(iframe, { defaultProps: { as: 'iframe', styles: styles.iframe } })}
+            {Box.create(iframe, {
+              defaultProps: {
+                as: 'iframe',
+                styles: styles.iframe,
+                style: { visibility: iframeLoaded ? 'visible' : 'hidden' },
```

Comment:
Move these styles into theme please.

Reply:
How do I access state in `EmbedStyles`?

Reply:
It will be available under props, use there a mix of types (actually it's already there):

![image](https://user-images.githubusercontent.com/14183168/65133466-f5321280-da02-11e9-8e43-5926471e09e1.png)